src/hooks/useActions.js

```diff
@@ -4,7 +4,7 @@ import { useWallet } from '../providers/Wallet'
 import { toHex } from 'web3-utils'
 import { getAppAddressByName } from '../lib/data-utils'
 
-const GAS_LIMIT = 350000
+const GAS_LIMIT = 420000
```

Comment:
given that with this chain we are having some issues regarding the gas limit, wondering if we should't use an ENV var to avoid deploying again on each gas issue that we might found

Reply:
I don't think an ENV var would help us skip the re deployment? Would make sense if we have different deployments for different chains tough.